tests/Unit/OAuth1/Service/AbstractServiceTest.php

```diff
@@ -212,4 +212,32 @@ public function testRequest()
 
         $this->assertSame('response!', $service->request('/my/awesome/path'));
     }
+
+    /**
+     * Test fails only in php5.3.
```

Comment:
The pull request also contains the fix, and the CI only ran against the
most recent commit. The test will just guard against any future regressions.
On 15 Oct 2015 23:44, "Elliot Chance" <notifications@github.com> wrote:

> In tests/Unit/OAuth1/Service/AbstractServiceTest.php
> <https://github.com/Lusitanian/PHPoAuthLib/pull/437#discussion_r42190656>:
>
> > @@ -212,4 +212,32 @@ public function testRequest()
> >
> >          $this->assertSame('response!', $service->request('/my/awesome/path'));
> >      }
> > +
> > +    /**
> > +     * Test fails only in php5.3.
>
> If it fails with PHP 5.3 why isn't the build failing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Lusitanian/PHPoAuthLib/pull/437/files#r42190656>.
>